### PR TITLE
add instructions for validation set

### PR DESCRIPTION
**Pull request by amj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -325,6 +325,54 @@ Additionally, you can follow along with the training progress with TensorBoard -
...
+
+### Validating on a different set of data
+
+This might be useful if you have some known set of 'good data' to test your
+netowrk against, e.g., a set of pro games.
+Assuming you've got a set of .sgfs with the proper komi & boardsizes, you'll
+want to preprocess them into the .tfrecord files, by running something similar
+to
+
+```
```
Comment (Kashomon) [inline README.md]: ` ```py `

Comment (Kashomon) [reply README.md]: (changed to python)

File: README.md [hunk 2]
```diff
@@ -325,6 +325,55 @@ Additionally, you can follow along with the training progress with TensorBoard -
...
+Assuming you've got a set of .sgfs with the proper komi & boardsizes, you'll
+want to preprocess them into the .tfrecord files, by running something similar
+to
+
+```python
+import preprocessing
+filenames = [generate a list of filenames here]
+for f in filenames:
+     try:
+         preprocessing.make_dataset_from_sgf(f, f + ".tfrecord.zz")
```
Comment (brilee) [inline README.md]: `f.replace('sgf', 'tfrecord.zz')`

File: README.md [hunk 3]
```diff
@@ -325,6 +325,55 @@ Additionally, you can follow along with the training progress with TensorBoard -
...
+easy as
+
+```
+python main.py validate path/to/validation/files/ --load-file=/path/to/model
+--logdir=path/to/tb/logs --num-steps=<number of positions to run validation on>
+```
+
+the `main.py validate` command will glob all the .tfrecord.zz files under the
+directories given as positional arguments and compute the validation error for
+`num_steps * TRAINING_BATCH_SIZE` positions from those files.
```
Comment (brilee) [inline README.md]: would be nice to abstract this out so you don't have to worry about training_batch_size. (I noticed it had been manually updated for 9s vs. 19s...)

File: README.md (hunk 2 above)
Comment (amj) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (amj) [reply README.md]: agreed.  It's also gonna be pretty specific to whatever accelerator/graphics card people are running this on.

**Comment by brilee** [comment]
/lgtm

The desire for a hyperparam solution continues to increase :P
